pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -558,10 +574,48 @@ func getDashbuilderTemplate(cr *api.KieApp, serversConfig []api.ServerTemplate,
 			dashbuilderTemplate.Cors = *cr.Status.Applied.Objects.Dashbuilder.Cors
 		}
 
+		// Enable EdgeTermination
+		if cr.Status.Applied.Objects.Dashbuilder.TerminationRoute != nil && len(cr.Status.Applied.Objects.Dashbuilder.TerminationRoute.Protocol) > 0 {
```

Comment:
Do we need to check protocol len there? As it is enum it should be always set and if not there should be used default value for it. Can it be dropped?
```suggestion
		if cr.Status.Applied.Objects.Dashbuilder.TerminationRoute != nil  {
```

Reply:
Was a old behaviour now we check if enableEdge value